lib/thumbs/pull_request_worker.rb

```diff
@@ -968,9 +1070,15 @@ def thumbot_merge
     end
 
     def forked_repo_branch_pr?
-      debug_message "pr.base.repo.full_name #{pr.base.repo.full_name}"
-      debug_message "pr.head.repo.full_name #{pr.head.repo.full_name}"
-      pr.base.repo.full_name != pr.head.repo.full_name ? true : false
+      debug_message "pull_request.base.repo.full_name #{pull_request.base.repo.full_name}"
+      debug_message "pull_request.head.repo.full_name #{pull_request.head.repo.full_name}"
+      pull_request.base.repo.full_name != pull_request.head.repo.full_name
+    end
+
+    def inspect
+      prefix = "#<#{self.class}:0x#{self.__id__.to_s(16)}"
```

Comment:
This is a bit overkill, but basically when using irb, the huge output of initializing this class was bothersome and the sheer noise made it not useful w/o probing the object to read whatever values the `inspect` should be making readily consumable.